docs/versioned-plugins/codecs/rubydebug-v3.0.3.asciidoc

```diff
@@ -0,0 +1,47 @@
+:plugin: rubydebug
+:type: codec
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v3.0.3
+:release_date: 2017-08-21
+:changelog_url: https://github.com/logstash-plugins/logstash-codec-rubydebug/blob/v3.0.4/CHANGELOG.md
+:include_path: ../include
+:logstash-ref: http://www.elastic.co/guide/en/logstash/6.0
```

Comment:
Is `logstash-ref` going to always point to the latest release? if so, this also needs to be templated.

Reply:
that's tbd. I've suggested adding the attribute to each doc file so that we aren't locked into having logstash-ref be the same value for every plugin.